templates/static/js/example.js

```diff
@@ -1,44 +1,22 @@
-/* global VANILLA_VERSION */
 (function () {
-  if (!window.VANILLA_VERSION) {
-    throw Error('VANILLA_VERSION not specified.');
-  }
-
-  var CODEPEN_PREFILL_CONFIG = {
-    title: 'Vanilla framework example',
-    head: "<meta name='viewport' content='width=device-width, initial-scale=1'>",
-    stylesheets: [
-      // link to latest Vanilla CSS
-      // if it's a demo, use local build.css for better QA, otherwise use latest published Vanilla
-      /demos\.haus$/.test(window.location.host)
-        ? `${window.location.origin}/static/build/css/build.css`
-        : 'https://assets.ubuntu.com/v1/vanilla-framework-version-' + VANILLA_VERSION + '.min.css',
-      // link to example stylesheet (to set margin on body)
-      'https://assets.ubuntu.com/v1/4653d9ba-example.css',
-    ],
-    tags: ['Vanilla framework'],
-  };
-
-  var CODEPEN_HEIGHT = 400;
-
   document.addEventListener('DOMContentLoaded', function () {
     var examples = document.querySelectorAll('.js-example');
 
     // IE11 doesn't support forEach on NodeList, CodePen doesn't support IE as well
     // so we don't want to load CodePen when forEach is not supported
     if (examples.forEach) {
-      examples.forEach(renderExample);
+      examples.forEach(fetchExample);
     }
```

Comment:
Do we still not want to support IE11 here? If we're not using CodePen, we can work around the NodeList/forEach issue with:
```
[].slice.call(examples).forEach(fetchExample(example))
```

Reply:
That's true, we can try to support IE here.

I did a quick look, and unfortuanely it's not just this `forEach` - I also use `element.closest()` for the dropdown functionality, which doesn't work in IE. And is a bit more annoying to polyfill than forEach.

So, I'd like to keep IE support out of this PR (just to replace CodePen that didn't have IE support either).
But we can have a look at IE support for this later. My worry is, that the experience will not be really nice anyway, and it may not be worth putting lots of work into it for the very small % of users we may have.

They always have a link to full example.

Reply:
Oh true, I didn't spot the `closest` - cool, was just a question, would be interesting to see what it looks like on IE and decide if we want to support it later.